data/language/en-GB.txt

```diff
@@ -3674,6 +3674,8 @@ STR_6352    :Density
 STR_6353    :{SMALLFONT}{BLACK}Low density
 STR_6354    :{SMALLFONT}{BLACK}Medium density
 STR_6355    :{SMALLFONT}{BLACK}High density
+STR_6356    :{SMALLFONT}{BLACK}Spawns ducks if park contains water
+STR_6357    :{SMALLFONT}{BLACK}Removes ducks if park contains ducks
```

Comment:
This seems a bit redundant to me. I would change it to this:

```suggestion
STR_6357    :{SMALLFONT}{BLACK}Removes all ducks from the map
```

Reply:
Indeed, that seems like a better tooltip.